snf-ui/app/index.html

```diff
@@ -20,6 +20,7 @@
   </head>
   <body class="{{content-for 'body-cls'}}">
     {{content-for 'body'}}
+    <script src="assets/ace/ace.js"></script>
```

Comment:
If we delete that tag we lose some functionalities of ace editor like highlighting